cypress/e2e/cloud/pinned.test.ts

```diff
@@ -1,6 +1,11 @@
 import {Organization} from '../../../src/types'
 
-describe('Pinned Items', () => {
+// This functionality is not currently on in cloud. These tests target
```

Comment:
Do we need to cross-reference this on any PR so that anyone working on the pinned items feature knows these tests are getting turned off?

Reply:
I don't believe there are any issues or PRs related to pinned items at the moment